helpers/kubernetes/cluster.go

```diff
@@ -260,6 +261,32 @@ func (c *Cluster) WaitForJobCompleted(namespace, jobName string, timeout time.Du
 	return wait.PollImmediate(time.Second, timeout, c.IsJobCompleted(client, jobName, namespace))
 }
 
+// IsDeploymentCompleted returns a condition function that indicates whether the given
+// Deployment is in Completed state or not.
+func (c *Cluster) IsDeploymentCompleted(deploymentName, namespace string) wait.ConditionFunc {
+	return func() (bool, error) {
+		deployment, err := c.Kubectl.AppsV1().Deployments(namespace).Get(context.Background(),
```

Comment:
I feel I should note that from what I saw, idiomatic Go uses shorter variable names, than it's normally the case in Epinio.

* https://github.com/golang/go/wiki/CodeReviewComments#variable-names
* https://talks.golang.org/2014/names.slide#19